qcengine/programs/nwchem/germinate.py

```diff
@@ -174,7 +175,8 @@ def muster_modelchem(method: str, derint: int, use_tce: bool) -> Tuple[str, Dict
         )
 
     elif method in _xc_functionals:
-        opts["dft__xc"] = method
+        if method != "dft":
+            opts["dft__xc"] = method
```

Comment:
A slight comment here, we may want to consider reordering this so that the `for` looks more like the following:
```python
elif method in _xc_functionals:
...
elif method == "dft":
....
```

This keeps the `_xc_functionals` list "pure" which may make a difference down the road. There is some duplicate `use_tce` commands however, but may be worth it.